src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall().toString());
```

Comment:
You can remove ".tostring" because you already define this in Ball class. When you use "getRandomBall" method you create Ball object and return it, so when you make  System.out.println( ballObject.tostring ) its the same as System.out.println( ballObject ).